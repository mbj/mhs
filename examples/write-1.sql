INSERT INTO test DEFAULT VALUES RETURNING id
