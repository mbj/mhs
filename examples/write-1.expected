INSERT INTO test DEFAULT VALUES RETURNING id
 id
----
  1
(1 row)

INSERT 0 1
